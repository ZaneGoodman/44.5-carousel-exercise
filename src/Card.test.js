import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";

//Smoke test
it("Renders Card component", () => {
  render(<Card />);
});
//Snapshot test
it("Card matches snapshot", () => {
  const { asFragment } = render(
    <Card caption="test1" src="test1Img" currNum={1} totalNum={1} />
  );
  expect(asFragment()).toMatchSnapshot();
});
