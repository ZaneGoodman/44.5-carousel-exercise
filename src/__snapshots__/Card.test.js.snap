// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      test1
    </h4>
    <img
      alt="test1"
      class="Card-image"
      src="test1Img"
    />
    <small
      class="Card-small"
    >
      Image 1 of 1.
    </small>
  </div>
</DocumentFragment>
`;
